imports/plugins/custom/in-app-wallet/lib/collections/schemas/accounts.js

```diff
@@ -0,0 +1,13 @@
+import { SimpleSchema } from "meteor/aldeed:simple-schema";
+import { registerSchema } from "@reactioncommerce/reaction-collections";
+import { accounts } from "/lib/collections/schemas/accounts";
+
+export const Accounts = new SimpleSchema([accounts, {
+  walletBalance: {
```

Comment:
You extend the accounts schema here. Very good. But why also modify the schema directly? See below